cooltools/api/expected.py

```diff
@@ -25,6 +25,26 @@
 _DIST = list(diag_expected_dtypes)[2]
 _NUM_VALID = list(diag_expected_dtypes)[3]
 
+# See notes in the docstring of agg_smooth_cvd()
```

Comment:
see notes in docstring of expected_cis? or also the per_region_smooth_cvd & genomewide_smooth_cvd?